### ISSUE TITLE
ANN: Meetup for SQuInTers and the BAJUs

### ISSUE THREAD

**Issue by i2000s** [JuliaQuantum/JuliaQuantum.github.io#15]
* Note: I use this issue post as an announcement since our Jekyll website is not authorized (@StefanKarpinski) to release for now and I don't yet have time to update the main website in the master branch in HTML... Registration for this event should go to this [online form](http://goo.gl/forms/T5qnGPndSE). 
* Update on 2/21/2015: Links to the slides and IPython notebooks preview for talks 1-2. Thanks.

During the [17th annual Southwest Quantum Information and Technology (SQuInT) workshop](http://physics.unm.edu/SQuInT/index.php) from Feb 19 to 21, 2015, JuliaQuantum organization will host a *breakout session* for the SQuInTers and the Bay area Julia users ***on Feb 21 (Saturday) from 7:30pm to 10pm*** in the ***Room Berkeley, DoubleTree Hilton Hotel***, 200 Marina Blvd., Berkeley, California, where the workshop is hosted. The location of the Berkeley room in the hotel can be found [here](http://doubletree3.hilton.com/resources/media/dt/JBKCADT/en_US/pdf/en_JBKCADT_Meeting_Factsheet_June12.pdf). It is on the second floor of the building. Beyond this *breakout session*, [Jarrett Revels](https://github.com/jrevels), one of the creators of JuliaQuantum, will present a *poster* (see #86 of the [abstract page](http://physics.unm.edu/SQuInT/abstracts.php)) on ***Feb 19*** in the workshop, and along with [Xiaodong Qi](https://github.com/i2000s)--another creator of this organization--will lead a *lunch gethering time* for SQuInTers who are interested in Julia and our projects on ***Feb 20*** in the lunch place of the workshop (our JuliaQuantum logo will show up on the lunch table).  

In the *breakout session* or the *meetup* on Feb 21 night, there will be three talks on the topic of *Scientific Applications of Julia* presented by ***[Philip Thomas](https://www.philipithomas.com/julia-jump/)*** from [StaffJoy](https://www.staffjoy.com/), ***[David Zeng](http://web.stanford.edu/~dzeng0/)*** and ***[Karanveer Mohan](http://web.stanford.edu/~kvmohan/)*** from [Stanford University](http://stanford.edu/~boyd/), and ***[Katharine Hyatt](http://web.physics.ucsb.edu/~kshyatt/)*** from University of California -- Santa Barbara on condensed matter theory. These talks are featured with optimization, quantum statistics and parallel computing in Julia, all of which heavily rely on numerics and could be applied to numerical simulations of quantum systems, optimal quantum control & metrology, quantum machine learning simulations and so on. In the mean time, these are also topics Julia is good at, and have been applied to a wide range of areas. In fact, JuliaQuantum is looking for the possibilities of interfacing with those well-developed packages and programs for our future projects of quantum libraries in Julia. In the talks, our invited speakers will demonstrate how easily Julia packages can be used to solve problems in those fields. This event is a trial to get people from classical computer science, mathematics and quantum science communities together. It could trigger a long-term collaborations crossing disciplines and inspire new ideas and development for the future of programming language and computer packages for the quantum era.  

Below is the detailed agenda for the Feb 21 breakout/meetup session. Limited refreshments and water will be served 15min before it starts.

## Agenda for Feb 21
* 7:30 Meetup starts with a brief introduction to the event, our sponsor & host organization--JuliaQuantum.

* 7:35-8:10 Talk 1: *Predictive Analysis in Julia - An overview of the JuMP package for optimization*
    
    Speaker: Philip Thomas from StaffJoy
    
    Abstract: This talk provides a brief overview of the JuMP package for optimization in Julia. 
JuMP provides an extensible tool for expressing and solving linear programming, integer programming, and convex programing problems. This talk will focus on expressing problems in the JuMP metalanguage, including a knapsack problem and constraint programming problem. A Lagrangian mechinics problem may be discussed as an example. 

    A preview of one example can be found [here](https://github.com/staffjoy/jump-examples). ***Vagrant*** is suggested to be installed, cloning the code, and running ```vagrant up``` to build the dev environment. The link to the slides is in the readme too.

    
* 8:15-8:50 Talk 2: *Convex.jl: Optimization for Everyone*

    Speakers: David Deng and Karanveer Mohan, possibly also with Jenny Hong
and Madeleine Udell from Stanford.

    Abstract: This talk will start with a brief overview of how the
Convex.jl package works and the types of problems it can solve, and
really showcase how convenient it is to use. It will be clear that
Convex.jl is easily usable by just about anyone for their basic
optimization needs. One or two more involved applications of using
Convex.jl to solve real world problems will be demonstrated from a good
pool of examples. Hopefully there will be an example on quantum tomography.

    Slides can be previewed at [here](http://web.stanford.edu/~dzeng0/Slides). There are also two interactive IPython notebook examples for the Demo section and can be previewed at [here](http://nbviewer.ipython.org/github/JuliaOpt/Convex.jl/blob/master/examples/time_series/time_series.ipynb) and [here](http://nbviewer.ipython.org/github/JuliaOpt/Convex.jl/blob/master/examples/tomography/tomography.ipynb).

* 8:50-9:25 Talk 3: *Quantum Statistical Simulations with Julia*

    Speaker: Katharine Hyatt from UCSB
    
    Abstract: Using computers to probe quantum systems is becoming more
and more common in condensed matter physics research. Many of the
commonly used languages and techniques in this space are either
difficult to learn or not performant. Julia has allowed us to quickly
develop and test codes for a variety of commonly used algorithms,
including exact diagonalization and quantum Monte Carlo. Its parallel
features, including some MPI and GPGPU integration, make it particularly
attractive for many quantum simulation problems. I'll discuss what
features of Julia have been most useful for us when working on these
simulations and the developments we're most excited about.

* 9:25-10:00pm Julia users interaction time as well as the Julia
developer business session time (at different zones of the room).

## Other notes regarding the breakout session
* Food bought outside of the hotel may not be allowed. 

* There may not be noticeable signs to indicate the meetup session. You can just ask the front desk for Room Berkeley if you cannot find us. 

* We will work out the vedios and presentation materials and share them freely online after the meetup--just in case you missed it.

* Seats are limited and it would be good to register through the following Google form. If the form does not show up on your browser, please use [this link](http://goo.gl/forms/T5qnGPndSE). Thanks. 
<iframe src="https://docs.google.com/forms/d/1siZ-gYL5-u3_sl_YnSa-qhjUtLxszcqBiwctf9CV15o/viewform?embedded=true" width="760" height="500" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>

**Comment by jrevels** [comment]
Woohoo! Great job putting this together!

**Comment by christianpeel** [comment]
I'm curious to know why there has not been a mention of it on 
     http://www.meetup.com/Bay-Area-Julia-Users/

**Comment by i2000s** [comment]
@christianpeel Just to clarify, I have sent the announcement over the weekend earlier to that meetup group, and requested another time this morning. No response so far. From an earlier communication with them, one of the organizer told me that they do not currently offer sponsorships to other events and wish us luck. I guess the organizers of the BAJU are hesitated to allow our event announced in the Bay area Julia users group as they are from /forio. But I am not entirely sure what is happening there, and I could be wrong on this matter. They may be just too busy to respond. 

The bottom line is that I don't have any admin power over the BAJU meetup group. I have put time and prepaid the expensive rental and service fees to the hotel out of my pocket for the Feb 21 meetup, and I am not going to make any money out of it in any sense. All of our invited speakers have put a lot of time to prepare for it, and are traveling on their own expenses as well. I hope more Julia users in the Bay area can benefit from it. I appreciate if you or anyone can send a message to the organizers of the BAJU on what you are thinking of.  Thanks.

**Comment by christianpeel** [comment]
@i2000s    Thanks for the detailed response!    I don't have any personal contact with the BAJU admins. I send emails through meetup.com to them earlier suggesting they talk with you.  They did not respond to me.   Hopefully they will get their act together and put your meetup on their calendar.

**Comment by i2000s** [comment]
@christianpeel Thank you for your action. We are using other social networks now. Hopefully we can reach out to people who are most interested in to join. Feel free to spread out our message.

**Comment by philipithomas** [comment]
I will be presenting about Julia for Mathematical Programming. 

My examples are here - if you want to follow along, I suggest installing Vagrant, cloning the code, and running `vagrant up` to build the dev environment. The link to my slides is in the readme too. 

https://github.com/staffjoy/jump-examples

**Comment by i2000s** [comment]
Links updated on the main post. Thanks.

**Comment by i2000s** [comment]
We just concluded the meetup! Thank you guys for coming. We had about 24 participants from the SQuInT workshop (majority), Berkeley Institute of Data Science, Stanford math dept and many other institutes. 

I will upload the video records and slides/demos online later.

**Comment by philipithomas** [comment]
@i2000s When do you think the videos will be online?
